frontend/public/components/_icon-and-text.scss

```diff
@@ -1,8 +1,14 @@
 .co-icon-and-text {
   align-items: baseline;
   display: flex;
+  min-width: 0; // required so co-break-word works
+  @include co-break-word;
```

Comment:
If we do this, we should get rid of `CamelCaseWrap`. But I'm not sure we really want to do this globally for `co-icon-and-text`.

Have we considered ellipses truncating the last column in tables where this happens? This is pretty rare, correct?